src/test/java/com/targomo/jackson/datatype/trove/ser/TIntFloatMapSerializerTest.java

```diff
@@ -0,0 +1,23 @@
+package com.targomo.jackson.datatype.trove.ser;
+
+import gnu.trove.map.TIntFloatMap;
+import gnu.trove.map.hash.TIntFloatHashMap;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class TIntFloatMapSerializerTest extends BaseTroveSerializerTest {
+
+    @Test
+    public void testSimple() throws IOException
+    {
+        TIntFloatMap map = new TIntFloatHashMap();
+        map.put(16, 1.5f);
```

Comment:
maybe also add on for the edge case: empty map?

Reply:
added the edge test as well